source/slang/slang-mangle.cpp

```diff
@@ -238,9 +238,23 @@ namespace Slang
         {
             emitRaw(context, "E");
         }
+        else if (const auto bottomType = dynamicCast<BottomType>(type))
+        {
+            emitRaw(context, "B");
+        }
+        else if (auto funcType = dynamicCast<FuncType>(type))
```

Comment:
Do we need to have a case for TupleType here?

Reply:
Not yet, they're used as a utility for parsing function types only at the moment so shouldn't ever get this far. It definitely couldn't hurt however.

Reply:
done